### PR TITLE
add longitude and latitude to UserGetDTO

### PR DESCRIPTION
**Pull request by anhuwi**

### CODE REVIEW

**Review by serainaschraff — APPROVED** [review]

**Review by daveking7 — COMMENTED** [review]
For the moment it is okay, but this is a big safety issue that evey user can get the exact location of every other user, since GET /users/userId only checks for general access.

**Comment by daveking7** [comment]
Created an issue for safety issue

**Comment by anhuwi** [comment]
> For the moment it is okay, but this is a big safety issue that evey user can get the exact location of every other user, since GET /users/userId only checks for general access.

sorry, did not noticed that get user is only general access. should i do a seprat get method onyl for lng and lat?

**Comment by daveking7** [comment]
> > For the moment it is okay, but this is a big safety issue that evey user can get the exact location of every other user, since GET /users/userId only checks for general access.
> 
> sorry, did not noticed that get user is only general access. should i do a seprat get method onyl for lng and lat?

Not needed anymore.

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/ChatControllerTest.java [hunk 1]
```diff
@@ -222,26 +222,34 @@ void createMessage_chatNotFound() throws Exception {
...
+        given(chatService.getUserIdFromOtherUser(chat1.getId(), user1.getId())).willReturn(user2.getId());
+        given(userService.getUserById(user2.getId())).willReturn(user2);
         MockHttpServletRequestBuilder getRequest = get("/users/{userId}/chats/{chatId}/newMsgs", user1.getId(), chat1.getId())
                 .contentType(MediaType.APPLICATION_JSON)
                 .header("authorization", user1.getToken());
         mockMvc.perform(getRequest).andExpect(status().isOk())
                 .andExpect(jsonPath("$[0].content", is("Hi")))
-                .andExpect(jsonPath("$[1].content", is("Bye")));
+                .andExpect(jsonPath("$[1].content", is("Bye")))
+                .andExpect(header().string("status", "OFFLINE"));
```
Comment (daveking7) [inline src/test/java/ch/uzh/ifi/hase/soprafs22/controller/ChatControllerTest.java:252]: @serainaschraff Should we test for the token header in UserController?

**Pull request merged**